bears/general/IndentationBear.py

```diff
@@ -199,30 +199,42 @@ def get_specified_block_range(self,
         close_pos = list(self.get_valid_sequences(
             file, close_specifier, annotation_dict))
 
-        to_match = len(open_pos) - 1
-        while to_match >= 0:
-            close_index = 0
-            while close_index < len(close_pos):
-                if(open_pos[to_match].position
-                   <= close_pos[close_index].position):
-                    ranges.append(
-                        SourceRange.from_absolute_position(
-                            filename,
-                            open_pos[to_match],
-                            close_pos[close_index]))
-                    close_pos.remove(close_pos[close_index])
-                    open_pos.remove(open_pos[to_match])
-                    to_match -= 1
-                    break
-                close_index += 1
-            if((len(close_pos) == 0 and to_match != -1) or
-               (len(close_pos) != 0 and to_match == -1)):
-                # None to specify unmatched indents
-                raise UnmatchedIndentError(open_specifier, close_specifier)
-
-        # Ranges are returned in the order of least nested to most nested
-        # and also on the basis of which come first
-        return tuple(ranges)[::-1]
+        if len(open_pos) != len(close_pos):
+            raise UnmatchedIndentError(open_specifier, close_specifier)
+
+        if open_pos == close_pos == []:
```

Comment:
well, you check their length in line 202 and therefore here you know they are of equal length. if `open_pos == []` it has length `0` right? Then the same must be True for `closed_pos` anyways.